app/models/filters/filter_base.rb

```diff
@@ -65,6 +65,7 @@ class FilterBase < ::ModelForm
     attribute :returned_to_homelessness_from_permanent_destination, Boolean, default: false
     attribute :creator_id, Integer, default: nil
     attribute :report_version, Symbol
+    attribute :inactivity_days, Integer, default: (365 * 2).to_i
```

Comment:
Not sure if I should be setting a default here or setting a default in the overview report or controller. From a user perspective, I think it's good if the dropdown auto-fills with "2 years" so that it's clear (which this does).

Reply:
I always prefer an explicit initializer when declaring a variable, so I think the default is a good idea.

Reply:
Is the `.to_i` necessary?  365*2 will return 730.

Reply:
no I guess not, i'll remove